wca-regulations.md

```diff
@@ -425,9 +425,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6i) Time penalties for stopping the solve are cumulative.
 - A7) Recording results:
     - A7a) The judge tells the competitor the result.
-        - A7a1) If the judge finds that the puzzle is solved, they call "OKAY".
-        - A7a2) If the judge assigns any penalties, they call "PENALTY".
-        - A7a3) If the result is DNF, the judge calls "DNF".
+        - A7a1) If the judge assigns any penalties or a DNF, they should tell the competitor.
```

Comment:
```suggestion
        - A7a1) The judge should indicate to the competitor that the puzzle is solved, or that a penalty will be applied.
```

Reply:
Im in strong favor of this change opposed to removing the judge indicating the puzzle is solved entirely. The reasoning is https://www.worldcubeassociation.org/regulations/#A6e